tst/teststandard/sort.tst

```diff
@@ -135,7 +135,7 @@ gap> for i in [0..100] do
 >      od;
 >    od;
 
-# Just sanity check I really am making string reps
+# Just consistency check I really am making string reps
```

Comment:
```suggestion
# Just check I really am making string reps
```

Reply:
Sorry @wilfwilson I didn't see your comment when I merged this. I agree with your suggestions and add a change for (using a completely different text) to https://github.com/gap-system/gap/pull/4082